actionview/lib/action_view/helpers/url_helper.rb

```diff
@@ -412,7 +412,7 @@ def link_to_unless(condition, name, options = {}, html_options = {}, &block)
       # Creates a link tag of the given +name+ using a URL created by the set of
       # +options+ if +condition+ is true, otherwise only the name is
       # returned. To specialize the default behavior, you can pass a block that
-      # accepts the name or the full argument list for +link_to_unless+ (see the examples
+      # accepts the name or the full argument list for +link_to_if+ (see the examples
```

Comment:
I think we can remove the "(see the examples in +link_to_unless+)" part because the doc have the example since f07d0e29da946aaa3956019365557e8712eb6817.

Reply:
Done.